plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -52,7 +52,8 @@ event BlockSubmitted(
 
  > ***Note**: For more information about parameter types, return types, valid values, and so on, see the [Plasma Contract API documentation](../contracts/BlockController.md#submitblock).*
  
-
+# Block retrieval
+To learn what blocks need to be retrieved during sync, compute block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, learn their commited hash values by reading `blocks` mapping.
```

Comment:
a couple of points:
 - how to get `childBlockInterval` and `nextChildBlock`
 - `commited hash` -> submitted root of the block Merkle tree (consistent with `# Block submission`)
 - `reading...mapping` -> I think this deserves listing out as a contract call, similar to [this for example](https://github.com/omisego/plasma-contracts/blob/46ccbd8f3a544c94955dd633d2c8ee7565e2bbbd/plasma_framework/docs/integration-docs/integration-doc.md#challenging-in-flight-transaction-output-exit)